prod/westeurope/internal/api/functions_assets_r3/function_app/terragrunt.hcl

```diff
@@ -90,14 +90,13 @@ inputs = {
 
     SLOT_TASK_HUBNAME = "ProductionTaskHub"
 
-    SERVICES_BY_SCOPE_BLOB_ID = "visible-services-by-scope.json"
-
   }
 
   app_settings_secrets = {
     key_vault_id = dependency.key_vault.outputs.id
     map = {
-
+      STATIC_WEB_ASSETS_ENDPOINT  = "common-STATIC-WEB-ASSETS-ENDPOINT"
+      STATIC_BLOB_ASSETS_ENDPOINT = "common-STATIC-BLOB-ASSETS-ENDPOINT"
```

Comment:
sorry I've just noticed these are put into the secrets section but these ain't secrets, they should be taken from the storage account dependencies

Reply:
fixed in [a408a97](https://github.com/pagopa/io-infrastructure-live-new/pull/264/commits/a408a978a5c979c65628462e82fa1eaf27f6bf3b)